server-spi-private/src/main/java/org/keycloak/exportimport/ImportProviderFactory.java

```diff
@@ -17,10 +17,21 @@
 
 package org.keycloak.exportimport;
 
+import java.util.Map;
+
+import org.keycloak.models.KeycloakSession;
 import org.keycloak.provider.ProviderFactory;
 
 /**
  * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
  */
 public interface ImportProviderFactory extends ProviderFactory<ImportProvider> {
+
+    ImportProvider create(KeycloakSession session, Map<String, String> overrides);
```

Comment:
As I've mentioned in a comment before, this should contain some Javadoc to describe the `overrides`. 

Can be done in a follow-up issue.